scene/2d/navigation_agent_2d.cpp

```diff
@@ -114,8 +114,53 @@ void NavigationAgent2D::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("link_reached", PropertyInfo(Variant::DICTIONARY, "details")));
 	ADD_SIGNAL(MethodInfo("navigation_finished"));
 	ADD_SIGNAL(MethodInfo("velocity_computed", PropertyInfo(Variant::VECTOR2, "safe_velocity")));
+
+#ifndef DISABLE_DEPRECATED
+	ClassDB::bind_method(D_METHOD("get_final_location"), &NavigationAgent2D::get_final_location);
```

Comment:
I think we shouldn't bind these older methods. Let code break, but avoid losing data due to properties changing.

Reply:
I was thinking of keeping them for just a few RC versions and removing them before Godot 4 release. That way users get a printed warning instead of a broken project, it makes it easier to adjust for them.

Reply:
well, removed as by akien's comment that we shouldn't add duplicated methods for comp.